### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
----

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ JabRef supports you in every step of your research work.
 - Fetch complete bibliographic information based on ISBN, DOI, PubMed-ID and arXiv-ID
 - Extract metadata from PDFs
 - [JabFox Firefox Add-on](https://addons.mozilla.org/en-US/firefox/addon/jabfox/) lets you import new references directly from the browser with one click
-	
+
```
Comment (koppor) [inline README.md]: Please no newlines with whitespace only

File: README.md [hunk 2]
```diff
@@ -43,18 +43,18 @@ JabRef supports you in every step of your research work.
 - Attach related documents: 20 different kinds of documents supported out of the box, completely customizable and extendable
 - Automatically rename and move associated documents according to customizable rules
 - Keep track of what you read: ranking, priority, printed, quality-assured 
-	
+
```
Comment (koppor) [inline README.md]: Please no newlines with whitespace only

File: README.md [hunk 3]
```diff
@@ -43,18 +43,18 @@ JabRef supports you in every step of your research work.
...
-	
+    
 #### Cite
 
 - Native [BibTeX] and [Biblatex] support
 - Cite-as-you-write functionality for external applications such as Emacs, Kile, LyX, Texmaker, TeXstudio, Vim and WinEdt.
 - Format references in one of the many thousand built-in citation styles or create your own style
 - Support for Word and LibreOffice/OpenOffice for inserting and formatting citations
-	
+
```
Comment (koppor) [inline README.md]: Please no newlines with whitespace only

File: README.md [hunk 4]
```diff
@@ -43,18 +43,18 @@ JabRef supports you in every step of your research work.
...
 - Cite-as-you-write functionality for external applications such as Emacs, Kile, LyX, Texmaker, TeXstudio, Vim and WinEdt.
 - Format references in one of the many thousand built-in citation styles or create your own style
 - Support for Word and LibreOffice/OpenOffice for inserting and formatting citations
-	
+    
 #### Share
 
 - Many built-in export options or create your own export format
-- Library is saved as a simple text file and thus it is easy to share with others via Dropbox and is version-control friendly
+- The library is saved as a simple text file and thus it is easy to share with others via Dropbox and is version-control friendly
```
Comment (koppor) [inline README.md]: I would write "The BibTeX/BibLaTeX library"

Comment (koppor) [reply README.md]: Maybe BibTex? - Many users do not know that we work with native text files. I wanted to express that here.

Comment (tobiasdiez) [reply README.md]: Maybe "bibliographic library" but in my opinion "library" should be clear enough.

**Comment by koppor** [comment]
Let's see whether @sakshamtaneja21 reacts or if we should just fix that word.

**Comment by koppor** [comment]
@sakshamtaneja21 if you come back to this PR, just reopen it. Until that, we will close it to have us not distracted from our other reviewing activities.

**Pull request closed without merging**